src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -24,47 +24,50 @@ def __init__(self, ivp, gaussfilt, with_smoothing):
 
         Notes
         -----
-        * gaussfilt.dynamicmodel contains the prior,
+        * gaussfilt.dynamic_model contains the prior,
         * gaussfilt.measurementmodel contains the information about the
         ODE right hand side function,
         * gaussfilt.initialdistribution contains the information about
         the initial values.
         """
-        if not issubclass(type(gaussfilt.dynamicmodel), ODEPrior):
-            raise ValueError("Please initialise a Gaussian filter with an ODEPrior")
+        if not issubclass(type(gaussfilt.dynamic_model), pnfs.statespace.Integrator):
```

Comment:
Formerly `ODEPrior` objects are `Integrator` objects now. Thus, they are in `filtsmooth` instead of `diffeq`. The interface is identical though.